### PR TITLE
Added flag for indicating if openmp was used to Cython submodules

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

**Comment by orbeckst** [comment]
Can you add a teeny-weeny test, at least for the serial case?

**Comment by richardjgowers** [comment]
I've added a test which requires that openmp be True, though this isn't a real requirement, I'm just curious what travis is doing.  I think ultimately it just needs to be either True or False

File: testsuite/MDAnalysisTests/test_distances.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import MDAnalysis.lib.distances
 
 import numpy as np
-from numpy.testing import (TestCase, raises,
+from numpy.testing import (TestCase, raises, assert_
```
Comment (orbeckst) [inline testsuite/MDAnalysisTests/test_distances.py]: comma missing after `_assert` !

File: testsuite/MDAnalysisTests/test_distances.py [hunk 2]
```diff
@@ -706,3 +706,9 @@ def test_missing_backend_raises_ValueError(self):
                                       backend="not_implemented_stuff")
         
         
+class TestOpenMP(object):
+    def test_serial(self):
+        assert_(MDAnalysis.lib._distances.OPENMP_ENABLED == False)
+
+    def test_serial(self):
```
Comment (orbeckst) [inline testsuite/MDAnalysisTests/test_distances.py]: really same name?

Comment (orbeckst) [inline testsuite/MDAnalysisTests/test_distances.py]: And yes, you could just test that it is either `True` or `False`.
Perhaps add a line to travis to explicitly echo what we are compiling by checking the flag.

**Comment by richardjgowers** [comment]
Ok it looks like travis is compiling openmp code.  I'll try and think of a nice way for the tests to tell you what they're actually testing

**Pull request merged**